### PR TITLE
test/librbd: add get_group test

### PR DESCRIPTION
**Pull request by songtongshuai**
Signed-off-by: songtongshuai_yewu <songtongshuai_yewu@cmss.chinamobile.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by songtongshuai** [comment]
@trociny Thanks for your advices, I modified the code and commit  again, but why it failed with  arm make check?

**Comment by trociny** [comment]
The arm make check failure is unrelated -- it looks like some issue with jenkins infrastructure. You may ignore it.

File: src/test/librbd/test_Groups.cc [hunk 1]
```diff
@@ -97,9 +97,15 @@ TEST_F(TestGroup, add_image)
   ASSERT_EQ(0, rbd_get_op_features(image, &op_features));
   ASSERT_TRUE((op_features & RBD_OPERATION_FEATURE_GROUP) == 0ULL);
 
+  rbd_group_info_t group_info;
+  ASSERT_EQ(0, rbd_get_group(image, &group_info, sizeof(group_info)));
```
Comment (trociny) [inline src/test/librbd/test_Groups.cc:101]: Ah, `rbd_get_group` for an image without group returns no error. But it sets `group_info.pool` to `RBD_GROUP_INVALID_POOL`. So could you please add the additional test:
```
ASSERT_EQ(group_info.pool, RBD_GROUP_INVALID_POOL);
```

And could you please add the same test (before `image.group_image_add`) to `add_imagePP`?

File: src/test/librbd/test_Groups.cc [hunk 2]
```diff
@@ -97,9 +97,15 @@ TEST_F(TestGroup, add_image)
...
+
   ASSERT_EQ(0, rbd_group_image_add(ioctx, group_name, ioctx,
                                    m_image_name.c_str()));
 
+  ASSERT_EQ(-ERANGE, rbd_get_group(image, &group_info, 0));
+  ASSERT_EQ(0, rbd_get_group(image, &group_info, sizeof(group_info)));
```
Comment (trociny) [inline src/test/librbd/test_Groups.cc:107]: Sorry, forgot to mention this in my previous review.

I would like we also check that `group_info` contains expected values `name` and `pool`, similarly how we check the result of ` rbd_group_image_list` below.

**Pull request closed without merging**